script/addRemoveBooks.js

```diff
@@ -0,0 +1,89 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const bookDiv = document.getElementById('display-books');
+let id = 0;
+
+class Book {
+  constructor(bookTitle, bookAuthor, bookId) {
+    this.bookTitle = bookTitle;
+    this.bookAuthor = bookAuthor;
+    this.bookId = bookId;
+  }
+
+  static addBook(newBook) {
+    const book = document.createElement('div');
+    book.classList.add('book-dynamic');
+    newBook.bookId = id;
+    book.innerHTML = ` <h4 class = "title-author">"${newBook.bookTitle}" by ${newBook.bookAuthor}</h4>
+        <button class="remove-button remove-btn" id="${id}">Remove</button><br>
+      `;
+    bookDiv.appendChild(book);
+    id += 1;
+  }
+
+  static displayBooks() {
+    const books = Book.getBooks();
+    books.forEach((newBook) => Book.addBook(newBook));
+  }
+
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static storeBook(newBook) {
+    const books = Book.getBooks();
+    books.unshift(newBook);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(bookId) {
```

Comment:
Your remove functionality is not working properly. It is not deleting the exact element I want it to. Kindly fix it.
If you need any further explanation, let me know on slack.